src/main/groovy/com/netgrif/application/engine/petrinet/domain/dataset/logic/action/ActionDelegate.groovy

```diff
@@ -779,32 +789,50 @@ class ActionDelegate {
         return userService.loggedUser
     }
 
-    void generatePDF(String transitionId, String fileFieldId) {
+    void generatePDF(Transition t, FileField field) {
+        this.generatePDF(t.getStringId(), field.importId)
+    }
+
+    String buildStoragePath(String outputFolder, String outputName, Case aCase, String fileFieldId) {
```

Comment:
This is duplicate implementation of the method `FileFieldValue.getPath` .

Reply:
This function is now removed and the suggested method is used.